app/views/posts/show.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1>List of Posts</h1>
-<h3>post#show</h3>
+
+<div class="post_page">
+	<div class="post">
+	<h3>Post #<%= @post.id %> 
```

Comment:
- [ ] I would also recommend you render the post title instead

  ```
  <h3><%= @post.title %>
  ```

Reply:
Well resolved 👍🏽